common/src/main/java/org/apache/sedona/common/raster/RasterAccessors.java

```diff
@@ -127,6 +128,70 @@ public static String getGeoReference(GridCoverage2D raster, String format) {
         }
     }
 
+
+    public static double[] getGeoTransform(GridCoverage2D raster) throws FactoryException {
+        // size of a pixel along the transformed i axis
+        double magnitudeI;
+
+        // size of a pixel along the transformed j axis
+        double magnitudeJ;
+
+        // angle by which the raster is rotated (Radians positive clockwise)
+        double thetaI;
+
+        // angle from transformed i axis to transformed j axis (Radians positive counter-clockwise)
+        double thetaIJ;
+
+        RenderedImage renderedImage = raster.getRenderedImage();
+
+        // x ordinate of the upper-left corner of the upper-left pixel
+        double offsetX = renderedImage.getTileGridXOffset();
+
+        // y ordinate of the upper-left corner of the upper-left pixel
+        double offsetY = renderedImage.getTileGridYOffset();
```

Comment:
This is not correct. `offsetX` and `offsetY` should be directly retrieved from the upperLeftX and upperLeftY components of the raster metadata.